dislash/slash_commands/slash_client.py

```diff
@@ -1025,7 +1045,11 @@ async def _cache_guild_commands(self):
     # Special waiter
     async def wait_for_button_click(self, check=None, timeout=None):
         if check is None:
-            check = lambda ctx: True
+            def check(inter):
+                if message.id != inter.message.id:
+                    return False
+                else:
+                    return True

```

Comment:
```suggestion
check = lambda ctx: True
```
This is absolutely wrong. `message` out of nowhere?? I think some of the code just migrated here due to the incorrect operation of the automatic parser of your IDE.

Reply:
but pep-8 say do not assign a lambda expression, use a def

Reply:
when ```suggestion
            def check(inter):
                return True
```

Reply:
I'm tired of explaining to you for the third time what is wrong here. I ask you to carefully reread my comments on GitHub and messages in Discord.